client/src/components/Suggestion/Suggestion.test.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme, { mount, shallow, configure } from "enzyme";
+import axios from "axios";
+import puppeteer from 'puppeteer';
+import {suggestionsData,teststate} from './testdata.js';
+import Suggestion from './Suggestion.jsx';
+import 'jest-styled-components';
+
+configure({adapter:new Adapter() });
+
+
+const testObj = {'listingId':11};
+
+expect.extend({
+    toBeWithinRange(received, floor, ceiling) {
+      const pass = received >= floor && received <= ceiling;
+      if (pass) {
+        return {
+          message: () =>
+            `expected ${received} not to be within range ${floor} - ${ceiling}`,
+          pass: true,
+        };
+      } else {
+        return {
+          message: () =>
+            `expected ${received} to be within range ${floor} - ${ceiling}`,
+          pass: false,
+        };
+      }
+    },
+  });
+
```

Comment:
is this a helper for testing a range of values for your state?  if so, could it be refactored to make use of Jest's .toBeGreaterThan/.toBeLessThan functions instead?

Reply:
yeah it's one of the helper function listed in the documentation.  Since the score should be in the range 1-5, thought it would be more appropriate to use that function.